orangecontrib/spectroscopy/io/neaspec.py

```diff
@@ -604,26 +588,29 @@ def read_spectra(self):
         df = self.resampled_df
         # format output to be used in the read method
 
-        self.original_df = []
+        # format data
         out_data = df.drop(columns=["Row", "Column", "Run", "Channel"]).values
         out_data = out_data.astype(np.float64)
         # formatting metas
         meta_domain = [
-            Orange.data.ContinuousVariable.make("column"),
-            Orange.data.ContinuousVariable.make("row"),
+            Orange.data.ContinuousVariable.make("map_x"),
+            Orange.data.ContinuousVariable.make("map_y"),
```

Comment:
When you rebase to get rid of the large file commit, please use `from orangecontrib.spectroscopy.utils import MAP_X_VAR, MAP_Y_VAR`  from #654